### ISSUE TITLE
about  the loss function

### ISSUE THREAD

**Issue by YongshengDong** [jiupinjia/Deep-adversarial-decomposition#9]
Hi,
Thanks for your good work!  But after reading the paper and the code, I'm confused about the loss function，
1. Where is DM？
2. I also can not find βC and βM which are in the final objective function formula in this code.

**Comment by jiupinjia** [comment]
Hi @YongshengDong, sorry for the late reply. 
1. If you look at the file unmix_gan.py, the DM1 and DM2 are define as `self.net_D1` and `self.net_D2` at line 29 and 30.
2. βC and βM are defined as `self.lambda_adv` in unmix_gan.py at line 60. If our method, we first set βC = βM = 0 for
the first 10 epochs to stabilized training and then set βC = βM = 0.001 for the rest epochs. Pls refer to Sec 3.4. Implementation Details of our paper for more details.
Thanks.

**Comment by YongshengDong** [comment]
Get it! Thanks
